parcels/field.py

```diff
@@ -143,7 +143,13 @@ class Field(object):
     """Class that encapsulates access to field data.
 
     :param name: Name of the field
-    :param data: 2D, 3D or 4D numpy array of field data
+    :param data: 2D, 3D or 4D numpy array of field data.
+           1. If data shape is [xdim, ydim], [xdim, ydim, zdim], [xdim, ydim, tdim] or [xdim, ydim, zdim, tdim],
+              whichever is relevant for the dataset,
+              use the flag transpose=True
```

Comment:
Why over three lines here? Can be two lines if you remove the return between dataset, and use